### DISCUSSION TITLE
Enable 2FA!!!

### DISCUSSION THREAD

**Discussion by mishmanners** [mishmanners/GitHub-Like-A-Boss#5] [General] [answered]
Who has enabled two-factor authentication???

**Comment by lythix** [comment, marked as answer]
I've done that. 

I am using the LastPass Authenticator... Its better than the Google one because it backs up all of your accounts in the cloud. The Google one doesn't (or didn't at the time. Ask me how I know...)

**Reply by surlydev** [reply]
I use Authy (which I believe is now owned by Twilio) and they also backup your stuff to the cloud, and if you use your mobile number, when you install the app on another device it gets all your stuff super simple. It is Google compatible.

**Reply by surlydev** [reply]
Another reason I didn't use the LastPass one any longer is because something happened and stopped working, and fortunately I was still logged in in the browser and was able to rectify the issue. How do you get into LastPass Authenticator if you lose your phone and have to start again With the Authy one, all you need is a SIM with the same number and then you are in Authy, then in there you have the 2FA to your LastPass, so you then have access to all your LastPass stored passwords.

**Reply by erwinheitzman** [reply]
You restore the backup on another phone

**Reply by mishmanners** [reply]
Awesome everyone!!! @mayakacz will be SOOOO proud of you all 😄

**Reply by surlydev** [reply]
I tried to put it on two phones @erwinheitzman I think that's where the issues started.

**Reply by mishmanners** [reply]
Yes don't put LastPass on multiple phones 😄

**Reply by surlydev** [reply]
@MishManners I have LastPass on my personal and work phones, and it works just fine. LastPass Authenticator should be possible to have on multiple devices. What about the 'bus factor' for 2FA devices ?

**Reply by mayakacz** [reply]
Can confirm, very proud 😊

**Comment by surlydev** [comment]
I'm concerned about enabling 2FA on GitHub because I know of a streamer who got distracted during the 2FA setup process, got the one-time-pad codes, but because he didn't confirm his details at the time something went wrong, and he could no longer log into his account. The codes did not work. GitHub support would not help him fix it - even though he uses the same stream name everywhere, and his avatar is his FACE.

**Reply by erwinheitzman** [reply]
This should be easy to reproduce with a dummy account if that is true. At which point you could report an issue with a reproducible example 😉

**Reply by surlydev** [reply]
I could try, I might set up another login account on my computer, too, just to make sure I'm isolated even further.

I'm concerned that they couldn't help him (like I said ,his face is his avatar, and he streams), he could have quite easily put a file in one of his repos as his computer was still logged in, and most concerning, they accepted an email from him to tell them to delete the account. If the email was compromised he might not notice that his GitHub account was going to be deleted.

I understand why 2FA is important, and do use it where I can. Maybe I will enable it on GitHub, and be super careful while doing it and make sure I complete (and test) the process.

**Reply by mishmanners** [reply]
If you're streaming, don't show passwords on screen 👍 and if there's any issues let us know. Just take your time setting up 2FA, and don't do it whilst distracted 😉

**Comment by erwinheitzman** [comment]
2FA is like coffee, you just need it 😄

**Reply by mishmanners** [reply]
YES!!! Except I don't drink coffee 😋

**Comment by determin1st** [comment]
hope it will be optional.. 8)

**Reply by mishmanners** [reply]
Yes 2FA is optional

**Reply by mishmanners** [reply]
BUT HIGHLY RECOMMENDED

**Reply by surlydev** [reply]
So is coffee

![image](https://user-images.githubusercontent.com/880671/82562494-3fa8f780-9b6d-11ea-81ac-21d49230b132.png)

**Comment by surlydev** [comment]
I enabled it. 

![image](https://user-images.githubusercontent.com/880671/82564059-06be5200-9b70-11ea-829b-08706a4783f4.png)
